modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -139,12 +133,12 @@ ifndef::tf-classic,tf-hcp[]
 endif::tf-classic,tf-hcp[]
 * Host prefix: /23
 +
-ifdef::rosa-hcp[]
+ifdef::openshift-rosa-hcp[]
 [NOTE]
 ====
 When using {hcp-title}, the static IP address `172.20.0.1` is reserved for the internal Kubernetes API address. The machine, pod, and service CIDRs ranges must not conflict with this IP address.
```

Comment:
Can we just remove "When using {hcp-title}"? We already know the customer will only be using ROSA HCP in this scenario.